examples/zone-printer/README.md

```diff
@@ -1,258 +1,244 @@
+for ctx in $(kubectl config get-contexts -o=name --kubeconfig clusters.yaml); do
   kubectl --context="${ctx}" create -f app/
 done
 ```
 
-## 2. Reserve a static IP on GCP.
+## Step 3: Reserve a static IP address
 
-We need to reserve a static IP on GCP for kubemci. Run the following
-command to reserve the IP.
+This command reserves a static IP address for the load balancer.
 
 ```shell
 ZP_KUBEMCI_IP="zp-kubemci-ip"
 gcloud compute addresses create --global "${ZP_KUBEMCI_IP}"
 ```
 
-## 3. Modify the Ingress manifest.
+## Step 4: Use the static IP on Ingress manifest
 
 Replace the value of `kubernetes.io/ingress.global-static-ip-name`
-annotation in `ingress/nginx.yaml` with the value of $ZP_KUBEMCI_IP.
+annotation in `ingress/nginx.yaml` with the value of `$ZP_KUBEMCI_IP`.
 
 ```shell
 sed -i -e "s/\$ZP_KUBEMCI_IP/${ZP_KUBEMCI_IP}/" ingress/nginx.yaml
 ```
 
-## 4. Deploy the Multi-Cluster Ingress (with kubemci)
+## Step 5: Deploy the multi-cluster Ingress with `kubemci`
 
-Run kubemci to create the multi-cluster ingress.
+Run `kubemci` to create the multi-cluster Ingress named `zone-printer`:
 
 ```shell
 kubemci create zone-printer \
     --ingress=ingress/nginx.yaml \
     --gcp-project=$PROJECT \
-    --kubeconfig=./zpkubeconfig
+    --kubeconfig=clusters.yaml
 ```
 
-Voila! You should have a multi-cluster ingress once the command
-successfully exits!
+This command creates the multi-cluster Ingress.
 
+## Step 6: View multi-cluster Ingress status
 
-## 5. Get the status of Multi-Cluster Ingress
-
-You can get the status of the multi-cluster ingress you just created
-by using `kubemci get-status` command.
+You can view the status of the multi-cluster Ingress you just created using the
+`kubemci get-status` command:
 
 ```shell
-kubemci get-status zone-printer --gcp-project=$PROJECT
+$ kubemci get-status zone-printer --gcp-project=$PROJECT
+
+Load balancer zone-printer has IPAddress 35.190.74.228 and
+is spread across 2 clusters (gke_yourproject_us-east4-a_cluster-1,
+gke_yourproject_europe-west1-c_cluster-2).
 ```
 
-## 6. Test the Multicluster Ingress
+## Step 7: Test the multi-cluster Ingress
 
-It may take several minutes for the ingress to propagate, but once it does, you
-can test it by curl'ing the IP from the get-status command. You should see it
-report a zone from the nearest region.
+It will take several minutes from when the Ingress and Google Cloud HTTP Load
+Balancer are created until they are ready. (You may see errors while the load
+balancer is getting ready.)
 
-## 7. Clean up
+Once the load balancer is ready to serve traffic, use the IP address printed
+in the previous step and query the server:
 
-Delete the multi-cluster ingress by using the `kubemci delete` command.
+```shell
+$ curl http://35.190.74.228
+
+Welcome to the global site! You are being served from us-central1-b.
+Congratulations
+```
+
+Multi-cluster Ingress has routed your traffic to the nearest GKE cluster
```

Comment:
routed your traffic to a cluster in nearest GCP region

Reply:
fixed